.github/workflows/fake-bpy-ci.yml

```diff
@@ -0,0 +1,84 @@
+name: fake-bpy-module CI
+
+on:
+  push:
+    branches:
+      # Temporarily trigger only on "ci_testing" branch
+      - ci_testing
+  # # Trigger on all PRs. By default, only run if activity type is opened, synchronize, or reopened
+  # pull_request:
+
+jobs:
+  build_modules:
+    name: Build fake-bpy modules
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+
+      - name: Blender downloads cache
+        uses: actions/cache@v2
+        id: blender-bin-cache
+        with:
+          path: blender-bin
+          key: blender-binaries-${{ runner.os }}-${{ hashFiles('**/tools/utils/download_blender.sh') }}
```

Comment:
Does it trigger to re-download when I change the Blender version `v2,82` to `v2.82a` in `download_blender.sh` script?

Reply:
It creates a hash of the file `download_blender.sh` and if the hash changes, it re-runs the script. This way, it is always guaranteed we are using the Blender artifacts which are intended by the script

Reply:
Note: The "cache" is purged if it is not used for more than 7 days from Github. So it might re-download even if the hash did not change. See here: https://github.com/actions/cache#cache-limits

Reply:
Thanks. I got it